### DISCUSSION TITLE
Thank You All

### DISCUSSION THREAD

**Discussion by Pridanc** [mriscoc/Ender3V2S1#1269] [General]
I've been using Professional software almost since I started my printing journey almost a year ago even if I'm not real sure how I got there.  

The other day I felt that I'd lost a stepper driver and thought what the heck, might as well install the BTT SKR Mini board if for no other reason that it has good reviews and has a couple of extras that look good on paper.   I was a bit hesitant at first to do so not knowing if I could continue with the same firmware so I took a look at the Professional MRISCOC GitHub to see what I could find.  Wow, low and behold I found the bin file for the board I had as well as with the BL-Touch included.  Winning!

Once I'd completed the install of the board, downloaded the firmware and placed it onto the Mini SD card, I set out to have the new Professional firmware install at first start up (probably not the best choice in hindsight but that is always 20/20 isn't it?)  and right off the bat something was wrong as the screen was not illuminating. Because of this I had no clue what, if anything, was happening with the new install on the new BTT board. 

In a panic it dawned on me that  the only true unknown was that I'd made a new cable to connect the board to the OE Creality Ender 3 v2 screen and with that in mind I quickly pulled the cable from the screen. Pulling the cable would not help the screen illuminate, but it might help a potential meltdown from incorrect wiring. I'd no idea if a failure was possible in this scenario but felt it couldn't hurt.  Low and behold once the cable had been pulled, suddenly, a myriad of flashing LED's on the SKR board were telling me that something was now happening, good, bad , or indifferent but was hoping for the good.

Without a screen or any other way to see inside the SKR board I was concerned but I remembered reading somewhere that once the new firmware bin file install was complete on the BTT board, that bin file would be renamed where it was on the Mini SD card.  And so it was!  Hallelujah,  maybe I was in luck.

Long story short is this: Once I corrected the wiring I'd done making the board-to-screen wire, all was absolutely golden. And I mean flipping perfect!  I'm printing away and couldn't be happier with my old dog of a printer. Since I do a decent amount of PP and TPU, who cares how slow this machine is right?

Anyway, HUGE thanks to all for creating such a wonderful firmware and experience for the non-technical like me.
